airlock/settings.py

```diff
@@ -402,3 +402,5 @@ def filter(self, record):  # pragma: no cover
         },
     },
 }
+
+SHOW_C3 = os.environ.get("SHOW_C3", "false") == "True"
```

Comment:
Because of the shared, global nature of env vars I'd be tempted to call the env var `AIRLOCK_SHOW_C3` just so it's obvious at a glance what system it relates to. I think the setting name can stay as it is.

As this is (hopefully) a short-lived setting it probably doesn't matter much either way, but just trying to spread the general convention around.

Reply:
Oh yes, I did that with the last one I added. will fix